File: parkingmanagements.client/src/app/core/models/payment.ts
export enum PaymentMethod {
  Cash = 0,
  Card = 1,
  Other = 2
}

export interface Payment {
  id: string;
  ticketId: string;
  amount: number;
  method: PaymentMethod;
  paidAt: Date;
  reference?: string;
}

export interface PaymentReq {
  amount: number;
  method: PaymentMethod;
  reference?: string;
}

File: ParkingManagements.Server/Data/Entities/Payment.cs
using ParkingManagements.Data.Entities;
using ParkingManagements.Data.Entities.Enums;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

public class Payment : BaseEntity
{
    public Guid TicketId { get; set; }
    public decimal Amount { get; set; }
    public PaymentMethod Method { get; set; }
    public DateTime PaidAt { get; set; }
    public virtual Ticket Ticket { get; set; }
}